articles/transfer-learning-with-deep-learning/index.md

```diff
@@ -0,0 +1,121 @@
+
+![hero](/engineering-education/transfer-learning-with-deep-learning/hero.jpg)
+[Image source](https://images.unsplash.com/photo-1569396116180-210c182bedb8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1049&q=80)
+
+Deep learning models need a lot of data. They may be trained on millions of data points before making predictions. This makes training very expensive. Furthermore, such models often perform only a single task. We look at how deep transfer learning can deal with these challenges.
+
+### Table of Contents
+
+1. Relationship between transfer learning and deep learning
+
+2. Transfer learning vs traditional machine learning
+
+3. When to use deep transfer learning
+
+4. Types of deep transfer learning
+
+### Prerequisites
+
+Basic understanding of machine learning and deep learning. For an introduction or refresher on some basic machine learning concepts check out this [article](/engineering-education/supervised-learning-algorithms/). Some machine learning and deep learning concepts are also covered [here](/engineering-education/automated-fake-news-detection/). For an introduction to transfer learning, I would advise going through my previous [article](https://www.section.io/engineering-education/basics-of-transfer-learning/).
+
+### Useful Terms
+
+**Pre-trained models** – models trained on a sizeable benchmark dataset to solve a problem similar to an already solved problem.
+
+**Marginal probability** – The probability of an event in the presence of all outcomes of another random variable.
```

Comment:
This needs some more clarity. Let's break it down. Marginal probability is the total probability of an RV given conditions on the other random variables. Correct? So what you could do is explain it from the context it will be used in this article.